src/components/standfirst.tsx

```diff
@@ -54,7 +54,7 @@ const thinHeadline = css`
 
 const immersive: SerializedStyles = css`
     ${styles}
-    ${headline.xsmall({ fontWeight: 'light' })}
+    ${headline.xxsmall({ fontWeight: 'light' })}
```

Comment:
Dotcom has `xsmall` for the standfirst.

Reply:
hmm it does, where did that come from because `frontend` uses 1.125rem whereas that would be 1.5rem 😅

Reply:
What was the conclusion on this?

Reply:
It seemed a little big to me at the time so thought about reducing it to the size on other articles. Happy to revert though.
![IMG_5301](https://user-images.githubusercontent.com/11618797/83163834-67670500-a103-11ea-8049-f7a0d56f8bd1.PNG)

Reply:
I'd strongly advocate for matching dotcom if we can, it makes it easier to share.

Reply:
Ok I've reverted that one. I guess as we gradually role article types out to users we can have some more general design reviews too